src/store.ts

```diff
@@ -159,7 +165,7 @@ export default class Store {
       'create table if not exists CVRs (id integer primary key autoincrement, batch_id integer references batches, filename text unique, cvr_json text)'
     )
     await this.dbRunAsync(
-      'create table if not exists elections (id integer primary key autoincrement, definition_json text)'
+      'create table if not exists configs (id integer primary key autoincrement, election_json text, is_test_mode boolean default 0)'
```

Comment:
I think you're going for the singleton row pattern here? I've found those to be tricky. Any reason not to use a key/value pair table for config, with unique key?

Reply:
That sounds reasonable to me.